Shifty.App/Components/Refunds.razor

```diff
@@ -0,0 +1,104 @@
+@namespace Components
+@using System.ComponentModel.DataAnnotations
+@using MudExtensions
+@using Shifty.App.Services
+@using Shifty.Api.Generated.AnalogCoreV1
+@using Shifty.Api.Generated.AnalogCoreV2
+@using Shared
+@using LanguageExt.UnsafeValueAccess
+@using Components
+@using Shifty.App.DomainModels
+@using System.Collections.ObjectModel
+@inject IPurchaseService PurchaseService
+@inject ISnackbar Snackbar
+
+<MudPaper Elevation="15" Style="border-radius: 5px; width:fit-content; min-width: 500px; margin:2rem auto">
+            @if(_purchasesFetched)
+            {
+                <MudIconButton Icon="@Icons.Material.Filled.ArrowBack" Color="Color.Primary" OnClick="@(() => _purchasesFetched = false)" />
+                <MudDataGrid
+                    T="Purchase"
+                    Items="@_purchases"
+                    EditMode="DataGridEditMode.Form"
```

Comment:
If all properties has `IsEditable="false"`, can we not just set the whole `DataGrid` as non-editable (not setting this line I believe, and possibly setting `ReadOnly="true"`)